src/main/java/jenkins/plugins/http_request/HttpRequest.java

```diff
@@ -451,11 +407,10 @@ public ListBoxModel doFillAuthenticationItems() {
         public FormValidation doCheckUrl(@QueryParameter String value)
                 throws IOException, ServletException {
             return FormValidation.ok();
-            // return HttpRequestValidation.checkUrl(value);
         }
 
-        public FormValidation doValidateKeyName(@QueryParameter String value) {
-            List<Authenticator> list = getAuthentications();
+        public static FormValidation doValidateKeyName(@QueryParameter String value) {
```

Comment:
@jglick  Speaking of form validation, what's the difference between `doValidate` and `doCheck`? Should I prefer one over the other for clarity?